FiegeIntegration/Overview/01_General.md

```diff
@@ -0,0 +1,27 @@
+# General information
+
+The *Fiege* driver seamlessly connects the *Actindo Core1 Platform* and the Fiege fulfillment system. Actindo sends automatically a dispatch note containing all relevant order and product data to Fiege when a delivery note has been issued by the *Order management* module. In turn, all shipment data provided by Fiege can be checked and monitored centrally in the *Fulfillment* module. 
```

Comment:
i'd add the info, that creating dispatch notes and triggering the export needs to be configured through the workflow as that does not "automagically" happen at any point - the export will then instruct the fiege driver to send it over to them